Library/Formula/pce.rb

```diff
@@ -7,8 +7,9 @@ class Pce < Formula
   head "git://git.hampa.ch/pce.git"
 
   devel do
-    url "http://www.hampa.ch/pub/pce/pre/pce-20140222-4b05f0c.tar.gz"
-    sha256 "44edaf071bb6840b6b3336174d528ff10c4dba8cb38194f0289fda81ac34f57f"
+    url "http://www.hampa.ch/pub/pce/pre/pce-20160308-72f1e10.tar.gz"
+    version "0.2.2+20160308"
```

Comment:
Even if upstream alternates their version scheme, we can always use the date.  Would that work?